File: tundra-cli/benchmarks/summing/sum.java
// File: sum_squares/SumSquares.java
// Compute sum of squares from 1..N (N = 2_000_000)

public class sum {
    public static long sum_squares(int n) {
        long total = 0L;
        int i = 1;
        while (i <= n) {
            long ii = (long)i * (long)i;
            total += ii;
            i++;
        }
        return total;
    }

    public static void main(String[] args) {
        int n = 3_000_000;
        long result = sum_squares(n);
        System.out.println(result);
    }
}

File: tundra-cli/benchmarks/summing/sum.js
// File: sum_squares/sum_squares.js
// Compute sum of squares from 1..N (N = 2_000_000)

function sum_squares(n) {
  let total = 0;
  let i = 1;
  while (i <= n) {
    const ii = i * i;
    total += ii;
    i += 1;
  }
  return total;
}

function main() {
  const n = 3000000;
  const result = sum_squares(n);
  console.log(result);
}

main();
